src/hostapi/oboe/pa_oboe.cpp

```diff
@@ -153,18 +152,20 @@ static signed long GetStreamWriteAvailable(PaStream *stream);
 static unsigned long GetApproximateLowBufferSize();
 
 // Commonly used parameters initialized.
-static unsigned long nativeBufferSize = 0;
-static unsigned numberOfBuffers = 2;
+static unsigned long g_nativeBufferSize = 0;
+static unsigned g_numberOfBuffers = 2;
 
 using namespace oboe;
 
 //Useful global variables
 int32_t g_inputDeviceId = kUnspecified;
 int32_t g_outputDeviceId = kUnspecified;
-PerformanceMode g_inputPerfMode = PerformanceMode::None;
-bool g_inputPerfModeUser = false;
-PerformanceMode g_outputPerfMode = PerformanceMode::None;
-bool g_outputPerfModeUser = false;
+
+PerformanceMode g_inputPerfMode = PerformanceMode::LowLatency;
```

Comment:
These non-static variables should be prefixed with paOboe to avoid collisions.
Also we try to avoid abbreviations.
For example paOboe_inputPerformanceMode